Source/ASDisplayNode+Layout.mm

```diff
@@ -1010,7 +1036,6 @@ - (void)_locked_setCalculatedDisplayNodeLayout:(const ASDisplayNodeLayout &)disp
   
   // Flatten the layout if it wasn't done before (@see -calculateLayoutThatFits:).
   if ([ASDisplayNode shouldStoreUnflattenedLayouts]) {
```

Comment:
We should just remove the entire if block here since callers of this method are now responsible for flattening the layout if need to.